README.md

```diff
@@ -14,6 +14,15 @@ plugins {
     id("com.github.gmazzo.buildconfig") version <current version>
 }
 
+// Make IDE recognize the generated BuildConfig file
+kotlin.sourceSets {
+    main {
+        kotlin.srcDirs(
+            file("$buildDir/generated/source/buildConfig/main"),
```

Comment:
You should avoid referencing static paths and use `Task`'s output's folder instead, to allow Gradle to correctly pick the dependency on it.

This plugin needs a modernization to conform latest Gradle standards. I'll probably be addressing this in there.

Actuallly,

Reply:
You are right! thanks for correcting me, fixed it now.